ktor-network/common/test/io/ktor/network/sockets/tests/UDPSocketTest.kt

```diff
@@ -67,8 +54,10 @@ class UDPSocketTest : CoroutineScope {
         assertTrue(socket.isClosed)
     }
 
+    // TODO: Test is failing with this exception:
+    //  io.ktor.utils.io.errors.PosixException.PosixErrnoException: POSIX error 49: Can't assign requested address (49)
```

Comment:
Note sure why this is failing

Reply:
Fixed: https://github.com/ktorio/ktor/pull/2129/commits/48758cfda04c0e1335121de8e4d548f57694111d